lib/shared/store.js

```diff
@@ -28,6 +28,52 @@ const fileExtensions = require('./file-extensions')
 const utils = require('./utils')
 const packageJSON = require('../../package.json')
 
+/**
+ * @summary Verify and throw if any state fields are nil
+ * @function
+ * @public
+ *
+ * @param {Object} object - state object
+ * @param {Array<Array<String>> | Array<String>} fields - array of object field paths
+ * @param {String} name - name of the state we're dealing with
+ * @throws
+ *
+ * @example
+ * const fields = [ 'type', 'percentage' ]
+ * verifyNoNilFields(action.data, fields)
```

Comment:
Should the example show the `name` parameter being used?

Reply:
Will submit a PR for this